### ISSUE TITLE
[Death Knight] [Talent] Killing Machine PPM implementation

### ISSUE THREAD

**Issue by Selot1234** [Stormforge-gg/Bugtracker-Fordring#1887]
**Description**: Killing Machine is expected to have 5 [PPM](https://wowwiki-archive.fandom.com/wiki/Procs_per_minute).

**Current behaviour**: Current implementation of PPM of KM is wrong: did testing on dummy: 
![kmproc](https://user-images.githubusercontent.com/38754622/182257083-5a6c84d2-1a7a-4c93-8bba-3cd749d0782b.jpg)
total hits: 3110, KM procs: 325, meaning i was at 325/3110=10,4% proc chance per hit.
i was using 2.7 weapon speed in both hands.
**Expected behaviour**: Let's do math for 2.7 weapon speed and 5 PPM mechanic (expected proc rate from hit):
PPH = 2,7 (Weapon speed) * 5 (PPM) / 60 = 22,5%. which means i should get around 0,225*3110=700 KM procs. 

in conclusion, now we are getting as twice as low procs as we should.

**Comment by Aithne99** [comment]
#1747

**Comment by Pinnen123** [comment]
What do you even mean. 49,5minutes, 5 procs per minute. 49,5*5=247,5. Proc chance per hit has nothing to do with it no?

**Comment by Pinnen123** [comment]
Dw brings abit more, but not double. And you're also counting total casts of melee, not actual hits, which makes your argument very doubtable

**Comment by Selot1234** [comment]
> #1747

can u give explanation why this is not bug in ur understanding? i clearly proved with math what's wrong and yet you're closing it with "Implausible at best."? Math is Mah you can't argue with it. Prove me my calculations are wrong.

@Pinnen123 you clearly don't know how PPM works and what is corelation between PPH, PPM and haste.

**Comment by Selot1234** [comment]
```
Every single source I can find says that 1/2/3/4/5 are the correct values, which means our implementation is correct and it was bugged on ATS
```
I'm not arguing there is wrong value set up in db, my point is current 5 PPM KM is not working as 5 PPM mechanic

**Comment by Dominiklas** [comment]
@Aithne99 pls take a look once again at calculations and my previous points
